spring-batch-excel/src/main/java/org/springframework/batch/item/excel/poi/PoiMultipleSheetExcelItemReader.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2006-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.batch.item.excel.poi;
+
+import org.springframework.batch.item.excel.ExcelFileParseException;
+
+/**
+ * @author Jyl-Cristoff
+ *
+ */
+public class PoiMultipleSheetExcelItemReader<T> extends
```

Comment:
Instead of subclassing `PoiItemReader` for this, wouldn't it be a better option to take an approach like the `MultiResourceItemReader` which is really just a decorator for another reader?